### ISSUE TITLE
💥 Crash Reported

### ISSUE THREAD

**Issue by sajanvox30** [codesandbox/codesandbox-client#8000]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | chrome | 116.0.0 | Windows 10           | 816c1afb3          |

  **Route:**
  https://codesandbox.io/s/hidden-snow-kdvyjg?file=/src/App.js

  **Error Code:**
4b76449eaaf0420383ce9c4977961769

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  TypeError: Cannot read properties of undefined (reading 'indexOf')
  TypeError: Cannot read properties of undefined (reading 'indexOf')
    at https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:714134
    at ua (https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:1865080)
    at xs (https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:1910162)
    at ml (https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:1895766)
    at hl (https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:1895691)
    at il (https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:1893039)
    at https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:1851481
    at https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:1926463
    at qo (https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:1851190)
    at $o (https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:1851426)
    at Yo (https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:1851361)
    at el (https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:1889828)
    at Object.enqueueSetState (https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:1856142)
    at n.setState (https://codesandbox.io/static/js/common.709bc413f.chunk.js?_sw-precache=40f1060820820f9fe696b2d7bea4e420:1:2109148)
    at n.value (https://codesandbox.io/static/js/default~app~embed.a4eae4441.chunk.js?_sw-precache=a86212056fd6968c6c5a148c575d3673:606:33196)
    at Object.<anonymous> (https://codesandbox.io/static/js/default~app~embed.a4eae4441.chunk.js?_sw-precache=a86212056fd6968c6c5a148c575d3673:606:31616)
    at https://codesandbox.io/static/js/default~app~embed~sandbox~sandbox-startup.9dd31f2a2.chunk.js?_sw-precache=8457aee172bec45db69698ff0e4aee8f:1:6731
    at Array.forEach (<anonymous>)
    at A (https://codesandbox.io/static/js/default~app~embed~sandbox~sandbox-startup.9dd31f2a2.chunk.js?_sw-precache=8457aee172bec45db69698ff0e4aee8f:1:6694)
    at P (https://codesandbox.io/static/js/default~app~embed~sandbox~sandbox-startup.9dd31f2a2.chunk.js?_sw-precache=8457aee172bec45db69698ff0e4aee8f:1:6903)
  
    in Unknown
    in bound 
    in div
    in li
    in Unknown
    in Unknown
    in ol
    in Unknown
    in he
    in t
    in Re
    in div
    in Styled(div)
    in t
    in WithTheme(t)
    in t
    in WithTheme(t)
    in div
    in Styled(div)
    in div
    in Styled(div)
    in v
    in t
    in div
    in Styled(div)
    in v
    in t
    in div
    in elements__Messages
    in div
    in elements__Container
    in n
    in WithTheme(n)
    in div
    in elements__ContentContainer
    in div
    in elements__Container
    in n
    in div
    in div
    in t
    in div
    in t
    in div
    in Ie
    in on
    in div
    in t
    in div
    in t
    in div
    in div
    in styled.div
    in div
    in elements__Container
    in Ie
    in Ay
    in Unknown
    in Unknown
    in t
    in t
    in div
    in elements__Content
    in n
    in withRouter(n)
    in div
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in Ie
    in c
    in t
  ```

  </details>

**Comment by CompuIves** [comment]
Hey! Thanks for reporting this, it looks like this happened because of an upgrade we did to the console-feed package. Whenever we had to print an Error without a stack-trace to the console, the component would crash. We've now reverted that update, so it should not happen again. In the upcoming weeks we're going to replace our console solution with a version that's powered by Chrome Devtools, so that will be a big upgrade when it comes to our logging experience, and will result in a more stable experience.

Apologies for the inconvenience, we've now pinned the version of console-feed in the meantime to make sure this won't happen again.
